ppr-ui/src/composables/mhrRegistration/useHomeOwners.ts

```diff
@@ -93,7 +103,7 @@ export function useHomeOwners (isMhrTransfer: boolean = false) {
       // Added second condition, because when an owner exists as a Sole Ownership, editing and clicking Done,
       // will change status to Tenants in Common unless above logic is in place..
       return HomeTenancyTypes.SOLE
-    } else if (numOfOwners > 1) {
+    } else if (numOfOwners > 1 && !hasETAError) {
```

Comment:
Just curious how this validation rule ties into determining the Home Tenancy Type. 
By the logic, it implies that if there is ever ever an `ETAError` that we always fall back to a `NA` type? 
This was a requirement?

Reply:
The only case where there can be an ETA and not have tenancy type of NA would be Sole Ownership. So I think I could remove the hasETAError and replace with hasETA, checking if an ETA is just present. Thanks for calling this out!